### PR TITLE
fix: minor grammar/typo fixes in comments

### PR DESCRIPTION
**Pull request by noamboy2006**
English grammer is too hard... lol

### CODE REVIEW

File: src/managers/base.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { Collection } from '../utils/collection.ts'
  */
 export class BaseManager<T, T2> {
   client: Client
-  /** Cache Name or Key used to differentiate caches */
+  /** Caches Name or Key used to differentiate caches */
```
Comment (DjDeveloperr) [inline src/managers/base.ts:11]: I think it was right there before.. lol or you mean "cache's"?

File: src/managers/channels.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export class ChannelsManager extends BaseManager<ChannelPayload, Channel> {
     return result
   }
 
-  /** Fetch a Channel by ID, cache it, resolve it */
+  /** Fetchs a Channel by ID, cache it, resolve it */
```
Comment (DjDeveloperr) [inline src/managers/channels.ts:46]: Fetches not fetchs

File: src/managers/emojis.ts [hunk 3]
```diff
@@ -20,7 +20,7 @@ export class EmojisManager extends BaseManager<EmojiPayload, Emoji> {
     return emoji
   }
 
-  /** Fetch an Emoji by Guild ID and Emoji ID, cache it and resolve it */
+  /** Fetchs an Emoji by Guild ID and Emoji ID, cache it and resolve it */
```
Comment (DjDeveloperr) [inline src/managers/emojis.ts:23]: Same typo here

File: src/structures/guild.ts [hunk 4]
```diff
@@ -288,7 +288,7 @@ export class Guild extends Base {
   }
 
   /**
-   * Fetch Guild's Integrations (Webhooks, Bots, etc.)
+   * Fetchs Guild's Integrations (Webhooks, Bots, etc.)
```
Comment (DjDeveloperr) [inline src/structures/guild.ts:291]: Typo 🤔

**Review by DjDeveloperr — CHANGES_REQUESTED** [review]

**Comment by DjDeveloperr** [comment]
Looks good, and please fix lint errors

File: src/managers/base.ts (hunk 1 above)
Comment (noamboy2006) [reply src/managers/base.ts:11]: I thought the word cache is used as a verb at the front, so I changed the from

File: src/managers/channels.ts (hunk 2 above)
Comment (noamboy2006) [reply src/managers/channels.ts:46]: oops

**Review by Helloyunho — APPROVED** [review]
I don't know English grammar well but I think it looks good!

**Review by DjDeveloperr — APPROVED** [review]

**Review by 6days9weeks — COMMENTED** [review]
looks good to me

**Pull request merged**